client/src/components/formHelpers/block-save-button.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Button } from '@freecodecamp/react-bootstrap';
 import { useTranslation } from 'react-i18next';
 
-function BlockSaveButton(props) {
+function BlockSaveButton(props: Record<any, unknown>) {
```

Comment:
```suggestion
function BlockSaveButton(props: Record<string, unknown>) {
```
I think all props will have `string` keys.